frontend/src/features/properties/hooks/useGeocoder.tsx

```diff
@@ -38,20 +40,44 @@ const useGeocoder = ({ formikRef }: IUseGeocoderProps) => {
   const [pidSelection, setPidSelection] = useState<IPidSelection>({ showPopup: false, geoPID: '' });
   const api = useApi();
   const dispatch = useDispatch();
-  const handleGeocoderChanges = async (data: IGeocoderResponse) => {
-    let parcelDataLayerResponse:
-      | Promise<FeatureCollection<Geometry, GeoJsonProperties>>
-      | undefined;
-    if (data.latitude && data.longitude) {
-      parcelDataLayerResponse = parcelsService.findOneWhereContains({
-        lat: data.latitude,
-        lng: data.longitude,
-      } as LatLng);
-      handleParcelDataLayerResponse(parcelDataLayerResponse, dispatch);
+
+  const updateForm = (newValues: any, nameSpace?: string) => {
+    if (!!formikRef?.current) {
+      // update form with values returned from geocoder
+      newValues.agencyId = formikRef.current.values.data.agencyId;
+      if (!!nameSpace) {
+        const { resetForm, values } = formikRef.current;
+        resetForm({ values: setIn(values, nameSpace, newValues) });
```

Comment:
set the initialvalues of this form to be the result of the search. that way only modifications to db stored values will register as changes. Also ensures that setting the pid/pin fields to the result of a search doesn't trigger another search.